website/examples/e-commerce/App.mobile.css

```diff
@@ -0,0 +1,253 @@
+@media (max-width: 899px) {
+  /* Filters overlay */
+
+  .container-filters {
+    background: #fff;
+    border-radius: 16px;
+    left: 0;
+    max-width: initial;
+    position: fixed;
+    top: 0;
+    transform: translateY(120vh);
+    transition: transform 300ms cubic-bezier(0.465, 0.183, 0.153, 0.946);
+    width: 100%;
+    will-change: transform;
+    z-index: 1;
+  }
+
+  .container-filters .container-body,
+  .container-filters .container-filters-footer {
+    opacity: 0;
+    transition: opacity 300ms ease-in;
+    transition-delay: 200ms;
+  }
+
+  .filtering .container-filters .container-body,
+  .filtering .container-filters .container-filters-footer {
+    opacity: 1;
+    transition-delay: 0;
+  }
+
+  .container-filters .container-header,
+  .container-filters .container-body {
+    padding: 2rem 2rem 0 2rem;
+  }
+
+  .filtering .header-logo {
+    left: 50%;
+    position: absolute;
+    top: 1.5rem;
+    transform: translateX(-50%);
+  }
+
+  .filtering .header-title {
+    display: none;
+  }
+
+  .filtering .container-filters {
+    position: absolute;
+    transform: translateY(4rem);
+  }
+
+  .filtering .container-results {
+    display: none;
+  }
+
+  .container-filters-footer {
+    background-color: #fff;
+    border-top: 1px solid #ebecf3;
+    bottom: 4rem;
+    display: flex;
+    justify-content: space-between;
+    padding: 1rem;
+    position: sticky;
```

Comment:
It looks like this didn't work.
![2019-06-21 13 03 44](https://user-images.githubusercontent.com/2982512/59918475-219aba80-9425-11e9-9a45-bdc3f1cdcb12.gif)